content/search.html

```diff
@@ -0,0 +1,119 @@
+---
+title: Search Results
+---
+
+<div id="main" class="span12" role="main">
+    <h1>Results</h1>
+</div>
+<div id="wrap" class="container" role="document">
+  <div id="responsive-menu"><button class="btn btn-primary" onclick="handleMenu()">Table of Contents</button></div>
+  <div id="content" class="row-fluid">
+    <div class="toc well span3 visible-toc">
+      <div style="display: flex; flex-direction: row; align-items: space-between; justify-content: space-between;">
+        <input type="text" id="search-input" placeholder="Search docs" /><br/>        
+        <button style="width: 40% margin: auto; margin-left: 5px;" class="btn-primary" onclick="handleSearch()">Search</button>
+      </div>
+      {% include "docs-toc.html" %}
+    </div>
+    <div id="main-docs" class="well span9 docs">
+      <h1>{{ title }}</h1>
+      {{ content | safe }}   
+    </div>
+    <div class="well span9 docs" id="search-results" style="display: none;">
+      <h3>Search Results</h3>
+      <div id="results-ul">
+
+      </div>
+    </div>
+  </div>
+</div>
+
+<script>
+    document.addEventListener("DOMContentLoaded", async () => {
+      //  Listen for enter event
+      const search = document.getElementById("search-input")
+      search.addEventListener("keyup", ({key}) => {
+        if (key === "Enter") {
+          handleSearch();
+        }
+      })
+      //  Get the search term from the url
+      const parsedSearchTerm = decodeURI(window.location.href.split("=")[1]);
```

Comment:
This seems like a bad and fragile way of getting a query string parameter. Maybe check out this article: https://stackoverflow.com/a/901144/55890